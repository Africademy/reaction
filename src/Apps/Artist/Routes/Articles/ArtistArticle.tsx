import { Sans, Serif } from "@artsy/palette"
import React, { SFC } from "react"
import { Box } from "Styleguide/Elements/Box"
import { Flex } from "Styleguide/Elements/Flex"
import { Col, Row } from "Styleguide/Elements/Grid"
import { Image } from "Styleguide/Elements/Image"
import { Separator } from "Styleguide/Elements/Separator"
import { Spacer } from "Styleguide/Elements/Spacer"
import { Responsive } from "Utils/Responsive"

interface ArticleItemProps {
  imageUrl: string
  date: string
  author: string
  title: string
  href: string
}

export const ArticleItem: SFC<ArticleItemProps> = props => {
  return (
    <Responsive>
      {({ xs }) => {
        if (xs) return <SmallArticleItem {...props} />
        else return <LargeArticleItem {...props} />
      }}
    </Responsive>
  )
}

const LargeArticleItem: SFC<ArticleItemProps> = props => {
  const { author, date, href, imageUrl, title } = props

  return (
    <React.Fragment>
      <Row>
        <Col sm={2}>
          <Date size="3" weight="medium">
            {date}
          </Date>
        </Col>
        <Col sm={8}>
          <Flex>
            <Box pr={12}>
              <Title size="5">
                <a href={href} className="noUnderline">
                  {title}
                </a>
              </Title>
              <Credit size="2" color="black60">
                {author}
              </Credit>
            </Box>
          </Flex>
        </Col>
        <Col sm={2}>
          <Image width="135px" height="85px" src={imageUrl} />
        </Col>
      </Row>

      {/* FIXME: Weird block height issue... */}
      <Spacer mt={2} />
      <Separator />
      <Spacer mt={3} />
    </React.Fragment>
  )
}

const SmallArticleItem: SFC<ArticleItemProps> = props => {
  const { author, date, href, imageUrl, title } = props

  return (
    <React.Fragment>
      <Flex justifyContent="space-between">
        <Box pr={3}>
          <Date size="1" weight="medium">
            {date}
          </Date>
          <Spacer mb={0.5} />
          <Title size="2">
            <a href={href} className="noUnderline">
              {title}
            </a>
          </Title>
          <Spacer mb={0.5} />
          <Credit size="1" color="black60">
            {author}
          </Credit>
        </Box>
        <Image width="80px" height="50px" src={imageUrl} />
      </Flex>

      {/* FIXME: Weird block height issue... */}
      <Spacer mt={2} />
      <Separator />
    </React.Fragment>
  )
}

const Date = Sans
const Title = Serif
const Credit = Serif
