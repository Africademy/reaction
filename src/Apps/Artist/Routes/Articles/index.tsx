import { Articles_artist } from "__generated__/Articles_artist.graphql"
import React from "react"
import { createFragmentContainer, graphql } from "react-relay"
import { ArticlesRefetchContainer as Articles } from "./ArticlesRefetchContainer"

export interface ArticlesRouteProps {
  artist: Articles_artist
}

export const ArticlesRoute: React.SFC<ArticlesRouteProps> = props => {
  return <Articles artist={props.artist as any} />
}

export const ArticlesRouteFragmentContainer = createFragmentContainer(
  ArticlesRoute,
  graphql`
    fragment Articles_artist on Artist {
      ...ArticlesRefetchContainer_artist
    }
  `
)
