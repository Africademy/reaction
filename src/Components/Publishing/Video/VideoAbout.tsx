import React, { Component } from "react"
import { Col, Row } from 'react-styled-flexboxgrid'
import styled from "styled-components"
import { pMedia } from "../../Helpers"
import { Fonts } from "../Fonts"
import { Text } from '../Sections/Text'

interface Props {
  article: any
}

export class VideoAbout extends Component<Props, null> {
  render() {
    const { article } = this.props
    const { media } = article
    return (
      <VideoAboutRow>
        <Col
          xs={12}
          sm={4}
          first="xs"
          last="sm"
        >
          <Title>
            Credits
          </Title>
          <Text layout="standard" html={media.credits} />
        </Col>

        <Col xs={12} sm={8} md={8} lg={8}>
          <Title>
            About the Film
          </Title>
          <Text layout="standard" html={media.description} />
        </Col>
      </VideoAboutRow>
    )
  }
}

const Title = styled.div`
  margin-bottom: 15px;
  ${Fonts.unica("s34")}
  ${pMedia.md`
    ${Fonts.unica("s32")}
  `}
`
const VideoAboutRow = styled(Row)`
  max-width: 1200px;
  margin: 60px auto;
`
