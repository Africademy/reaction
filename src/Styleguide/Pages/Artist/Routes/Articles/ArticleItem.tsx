import { Sans, Serif } from "@artsy/palette"
import React from "react"
import { Box } from "Styleguide/Elements/Box"
import { Flex } from "Styleguide/Elements/Flex"
import { Col, Row } from "Styleguide/Elements/Grid"
import { Image } from "Styleguide/Elements/Image"
import { Responsive } from "Styleguide/Utils/Responsive"

interface ArticleItemProps {
  imageUrl: string
  date: string
  author: string
  title: string
  href: string
}

export const ArticleItem = (props: ArticleItemProps) => {
  return (
    <Responsive>
      {({ xs }) => {
        if (xs) return <SmallArticleItem {...props} />
        else return <LargeArticleItem {...props} />
      }}
    </Responsive>
  )
}

const LargeArticleItem = (props: ArticleItemProps) => {
  return (
    <React.Fragment>
      <Row>
        <Col sm={2}>
          <Date size="3" weight="medium">
            {props.date}
          </Date>
        </Col>
        <Col sm={8}>
          <Flex>
            <Box pr={12}>
              <Title size="5">
                <a href={props.href} className="noUnderline">
                  {props.title}
                </a>
              </Title>
              <Credit size="2" color="black60">
                {props.author}
              </Credit>
            </Box>
          </Flex>
        </Col>
        <Col sm={2}>
          <Image width="135px" height="85px" src={props.imageUrl} />
        </Col>
      </Row>
    </React.Fragment>
  )
}

const SmallArticleItem = (props: ArticleItemProps) => {
  return (
    <React.Fragment>
      <Row>
        <Col sm={2}>
          <Box mb={0.5}>
            <Date size="1" weight="medium">
              {props.date}
            </Date>
          </Box>
        </Col>
        <Col sm={8}>
          <Flex>
            <Box pr={3}>
              <Title size="2">
                <a href={props.href} className="noUnderline">
                  {props.title}
                </a>
              </Title>
              <Credit size="2" color="black60">
                {props.author}
              </Credit>
            </Box>
            <Image width="80px" height="50px" src={props.imageUrl} />
          </Flex>
        </Col>
      </Row>
    </React.Fragment>
  )
}

const Date = Sans
const Title = Serif
const Credit = Serif
