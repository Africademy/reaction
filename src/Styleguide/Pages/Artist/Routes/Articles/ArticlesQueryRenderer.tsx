import { ContextConsumer, ContextProps } from "Components/Artsy"
import React from "react"
import { graphql, QueryRenderer } from "react-relay"
import { ArticlesRefetchContainer } from "./ArticlesRefetchContainer"

interface Props extends ContextProps {
  artistID: string
}

const PAGE_SIZE = 10

export const ArticlesQueryRenderer = ContextConsumer(
  class extends React.Component<Props> {
    render() {
      const { artistID, relayEnvironment } = this.props
      return (
        <QueryRenderer
          environment={relayEnvironment}
          query={graphql`
            query ArticlesQueryRendererQuery($artistID: String!, $first: Int!) {
              artist(id: $artistID) {
                ...ArticlesRefetchContainer_artist @arguments(first: $first)
              }
            }
          `}
          variables={{ artistID, first: PAGE_SIZE }}
          render={({ props }) => {
            if (props) {
              return <ArticlesRefetchContainer artist={props.artist} />
            } else {
              return null
            }
          }}
        />
      )
    }
  }
)
