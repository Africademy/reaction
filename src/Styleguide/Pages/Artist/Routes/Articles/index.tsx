import React from "react"
import { ArticlesRefetchContainer } from "./ArticlesRefetchContainer"

interface Props {
  artist: any
}

export class ArticlesRoute extends React.Component<Props> {
  render() {
    return <ArticlesRefetchContainer artist={this.props.artist} />
  }
}
