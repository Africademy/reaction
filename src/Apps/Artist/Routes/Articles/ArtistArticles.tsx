import { ArtistArticles_artist } from "__generated__/ArtistArticles_artist.graphql"
import React, { Component } from "react"
import { createRefetchContainer, graphql, RelayRefetchProp } from "react-relay"
import { PaginationFragmentContainer as Pagination } from "Styleguide/Components/Pagination"
import { Flex } from "Styleguide/Elements/Flex"
import { Col, Row } from "Styleguide/Elements/Grid"
import { ArticleItem } from "./ArtistArticle"

import {
  LoadingArea,
  LoadingAreaState,
} from "Apps/Artist/Components/LoadingArea"

const PAGE_SIZE = 10

interface ArtistArticlesProps {
  relay: RelayRefetchProp
  artist: ArtistArticles_artist
}

export class ArtistArticles extends Component<
  ArtistArticlesProps,
  LoadingAreaState
> {
  state = {
    isLoading: false,
  }

  loadNext = () => {
    const {
      artist: {
        articlesConnection: {
          pageInfo: { hasNextPage, endCursor },
        },
      },
    } = this.props

    if (hasNextPage) {
      this.loadAfter(endCursor)
    }
  }

  loadAfter = cursor => {
    this.toggleLoading(true)

    this.props.relay.refetch(
      {
        first: PAGE_SIZE,
        after: cursor,
        artistID: this.props.artist.id,
        before: null,
        last: null,
      },
      null,
      error => {
        this.toggleLoading(false)

        if (error) {
          console.error(error)
        }
      }
    )
  }

  toggleLoading = isLoading => {
    this.setState({
      isLoading,
    })
  }

  render() {
    return (
      <React.Fragment>
        <span id="jumpto--artistArticles" />

        <Row>
          <Col>
            <LoadingArea isLoading={this.state.isLoading}>
              {this.props.artist.articlesConnection.edges.map(
                ({ node }, index) => {
                  return (
                    <ArticleItem
                      title={node.thumbnail_title}
                      imageUrl={node.thumbnail_image.resized.url}
                      date={node.published_at}
                      author={node.author.name}
                      href={node.href}
                      key={index}
                    />
                  )
                }
              )}
            </LoadingArea>
          </Col>
        </Row>
        <Row>
          <Col>
            <Flex mb={2} justifyContent="flex-end">
              <Pagination
                pageCursors={
                  this.props.artist.articlesConnection.pageCursors as any
                }
                onClick={this.loadAfter}
                onNext={this.loadNext}
                scrollTo="#jumpto-RouteTabs"
              />
            </Flex>
          </Col>
        </Row>
      </React.Fragment>
    )
  }
}

export const ArtistArticlesRefetchContainer = createRefetchContainer(
  ArtistArticles,
  {
    artist: graphql`
      fragment ArtistArticles_artist on Artist
        @argumentDefinitions(
          first: { type: "Int", defaultValue: 10 }
          last: { type: "Int" }
          after: { type: "String" }
          before: { type: "String" }
        ) {
        id
        articlesConnection(
          first: $first
          after: $after
          before: $before
          last: $last
          sort: PUBLISHED_AT_DESC
          in_editorial_feed: true
        ) {
          pageInfo {
            hasNextPage
            endCursor
          }
          pageCursors {
            ...Pagination_pageCursors
          }
          edges {
            node {
              href
              thumbnail_title
              author {
                name
              }
              published_at(format: "MMM d, YYYY")
              thumbnail_image {
                resized(width: 300) {
                  url
                }
              }
              href
            }
          }
        }
      }
    `,
  },
  graphql`
    query ArtistArticlesQuery(
      $first: Int
      $last: Int
      $after: String
      $before: String
      $artistID: String!
    ) {
      artist(id: $artistID) {
        ...ArtistArticles_artist
          @arguments(first: $first, last: $last, after: $after, before: $before)
      }
    }
  `
)
