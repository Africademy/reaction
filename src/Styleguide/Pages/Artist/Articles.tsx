import React from "react"
import {
  graphql,
  QueryRenderer,
  createRefetchContainer,
  RelayRefetchProp,
} from "react-relay"
import { ContextProps, ContextConsumer } from "Components/Artsy"
import styled from "styled-components"
import { Col, Row } from "Styleguide/Elements/Grid"
import { Box } from "Styleguide/Elements/Box"
import { Flex } from "Styleguide/Elements/Flex"
import { Separator } from "Styleguide/Elements/Separator"
import { ArticleItem } from "./ArticleItem"
import { Pagination } from "Styleguide/Components/Pagination"

import { Articles_artist } from "__generated__/Articles_artist.graphql"

interface ArticlesProps {
  relay: RelayRefetchProp
  artist: Articles_artist
}

const PAGE_SIZE = 10

// Renders list of articles, we'll use this in our refetch container.
class ArticlesContainer extends React.Component<ArticlesProps> {
  loadPrev = () => {
    const {
      startCursor,
      hasPreviousPage,
    } = this.props.artist.articlesConnection.pageInfo
    if (hasPreviousPage) {
      this.loadBefore(startCursor)
    }
  }

  loadNext = () => {
    const {
      hasNextPage,
      endCursor,
    } = this.props.artist.articlesConnection.pageInfo
    if (hasNextPage) {
      this.loadAfter(endCursor)
    }
  }

  loadBefore(cursor) {
    this.props.relay.refetch(
      {
        first: null,
        before: cursor,
        artistID: this.props.artist.id,
        after: null,
        last: PAGE_SIZE,
      },
      null,
      error => {
        if (error) {
          console.error(error)
        }
      }
    )
  }

  loadAfter = cursor => {
    this.props.relay.refetch(
      {
        first: PAGE_SIZE,
        after: cursor,
        artistID: this.props.artist.id,
        before: null,
        last: null,
      },
      null,
      error => {
        if (error) {
          console.error(error)
        }
      }
    )
  }

  renderPagination() {
    return (
      <div>
        <Pagination
          {...this.props.artist.articlesConnection.pageCursors}
          onClick={this.loadAfter}
          onNext={this.loadNext}
          onPrev={this.loadPrev}
        />
      </div>
    )
  }

  render() {
    return (
      <div>
        <Row>
          <Col>
            <ArticleList>
              {this.props.artist.articlesConnection.edges.map(({ node }) => {
                return (
                  <ArticleItem
                    title={node.thumbnail_title}
                    imageUrl={node.thumbnail_image.resized.url}
                    date={node.published_at}
                    author={node.author.name}
                    href={node.href}
                  />
                )
              })}
            </ArticleList>
          </Col>
        </Row>

        <Box my={2}>
          <Separator />
        </Box>

        <Row>
          <Col>
            <Flex mb={2} justifyContent="flex-end">
              {this.renderPagination()}
            </Flex>
          </Col>
        </Row>
      </div>
    )
  }
}

// This is the actual Refetch Container we want to use.
const ArticlesRefetchContainer = createRefetchContainer(
  ArticlesContainer,
  {
    artist: graphql`
      fragment Articles_artist on Artist
        @argumentDefinitions(
          first: { type: "Int" }
          last: { type: "Int" }
          after: { type: "String" }
          before: { type: "String" }
        ) {
        id
        articlesConnection(
          first: $first
          after: $after
          before: $before
          last: $last
          sort: PUBLISHED_AT_DESC
          in_editorial_feed: true
        ) {
          pageInfo {
            hasNextPage
            hasPreviousPage
            startCursor
            endCursor
          }
          pageCursors {
            around {
              cursor
              page
              isCurrent
            }
            first {
              cursor
              page
              isCurrent
            }
            last {
              cursor
              page
              isCurrent
            }
          }
          edges {
            node {
              href
              thumbnail_title
              author {
                name
              }
              published_at(format: "MMM d, YYYY")
              thumbnail_image {
                resized(width: 300) {
                  url
                }
              }
              href
            }
          }
        }
      }
    `,
  },
  graphql`
    query ArticlesQuery(
      $first: Int
      $last: Int
      $after: String
      $before: String
      $artistID: String!
    ) {
      artist(id: $artistID) {
        ...Articles_artist
          @arguments(first: $first, last: $last, after: $after, before: $before)
      }
    }
  `
)

interface Props extends ContextProps {
  artistID: string
}

class ArticlesQueryRenderer extends React.Component<Props> {
  render() {
    const { artistID, relayEnvironment } = this.props
    return (
      <QueryRenderer
        environment={relayEnvironment}
        query={graphql`
          query ArticlesIndexQuery($artistID: String!, $first: Int!) {
            artist(id: $artistID) {
              ...Articles_artist @arguments(first: $first)
            }
          }
        `}
        variables={{ artistID, first: PAGE_SIZE }}
        render={({ props }) => {
          if (props) {
            return <ArticlesRefetchContainer artist={props.artist} />
          } else {
            return null
          }
        }}
      />
    )
  }
}

export const ArticlesContent = ContextConsumer(ArticlesQueryRenderer)

const ArticleList = styled.div``
