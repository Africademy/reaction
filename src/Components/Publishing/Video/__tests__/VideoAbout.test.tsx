// import { mount } from "enzyme"
import "jest-styled-components"
import React from "react"
import renderer from "react-test-renderer"
import { VideoArticle } from "../../Fixtures/Articles"
import { VideoAbout } from "../VideoAbout"

describe("Video About", () => {
  it("matches the snapshot", () => {
    const videoAbout = renderer.create(
      <VideoAbout
        article={VideoArticle}
      />
    ).toJSON()
    expect(videoAbout).toMatchSnapshot()
  })

  // it("renders the credits section", () => {
  //   const component = mount(
  //     <VideoAbout
  //       article={VideoArticle}
  //     />
  //   )
  //   // expect(toggleMute).toBeCalled()
  // })

  // it("renders the about section", () => {
  //   const component = mount(
  //     <VideoAbout
  //       article={VideoArticle}
  //     />
  //   )
  // })
})
