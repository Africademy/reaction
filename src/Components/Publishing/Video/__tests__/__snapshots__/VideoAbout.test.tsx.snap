// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video About matches the snapshot 1`] = `
.c0 {
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 0 1 auto;
  -ms-flex: 0 1 auto;
  flex: 0 1 auto;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  margin-right: -0.5rem;
  margin-left: -0.5rem;
}

.c1 {
  box-sizing: border-box;
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
  padding-right: 0.5rem;
  padding-left: 0.5rem;
}

.c4 {
  box-sizing: border-box;
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
  padding-right: 0.5rem;
  padding-left: 0.5rem;
}

.c3 {
  position: relative;
  width: 100%;
}

.c3 a {
  color: black;
  text-decoration: none;
  position: relative;
  background-image: linear-gradient(to bottom,transparent 0,#333 1px,transparent 0);
  background-size: 1.25px 4px;
  background-repeat: repeat-x;
  background-position: bottom;
}

.c3 a:hover {
  color: #999;
}

.c3 p,
.c3 ul,
.c3 ol {
  font-family: 'Adobe Garamond W08', 'adobe-garamond-pro', 'AGaramondPro-Regular', 'Times New Roman', 'Times', 'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 23px;
  line-height: 1.5em;
  padding-top: 1em;
  padding-bottom: 1em;
  margin: 0;
  font-style: inherit;
}

.c3 p:first-child {
  padding-top: 0;
}

.c3 p:last-child {
  padding-bottom: 0;
}

.c3 ul,
.c3 ol {
  padding-left: 1em;
}

.c3 li {
  font-family: 'Adobe Garamond W08', 'adobe-garamond-pro', 'AGaramondPro-Regular', 'Times New Roman', 'Times', 'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 23px;
  line-height: 1.5em;
  padding-top: .5em;
  padding-bottom: .5em;
}

.c3 h1 {
  font-family: Unica77LLWebRegular , 'Arial', 'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 40px;
  line-height: 1.1em;
  font-weight: normal;
  padding-top: 107px;
  padding-bottom: 46px;
  margin: 0;
  position: relative;
  text-align: center;
}

.c3 h1:before {
  content: "";
  width: 15px;
  height: 15px;
  background: black;
  border-radius: 50%;
  position: absolute;
  top: 69px;
  right: calc(50% - 7.5px);
}

.c3 h2 {
  font-family: Unica77LLWebRegular , 'Arial', 'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 32px;
  line-height: 1.1em;
  font-weight: normal;
  margin: 0;
}

.c3 h2 a {
  background-size: 1.25px 1px;
}

.c3 h3 {
  font-family: Unica77LLWebRegular , 'Arial', 'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 19px;
  line-height: 1.5em;
  font-weight: normal;
  padding-top: 23px;
  margin: 0;
}

.c3 h3 strong {
  font-weight: normal;
  font-family: Unica77LLWebMedium , 'Arial', 'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 19px;
  line-height: 1.5em;
}

.c3 h3 a {
  background-size: 1.25px 1px;
}

.c3 blockquote {
  font-family: 'Adobe Garamond W08', 'adobe-garamond-pro', 'AGaramondPro-Regular', 'Times New Roman', 'Times', 'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 40px;
  line-height: 1.1em;
  text-align: left;
  font-weight: normal;
  padding-top: 46px;
  padding-bottom: 46px;
  margin: 0;
  word-break: break-word;
}

.c3 .content-end {
  display: inline-block;
  content: "";
  width: 12px;
  height: 12px;
  background: black;
  border-radius: 50%;
  margin-left: 12px;
}

.c3 .artist-follow {
  vertical-align: middle;
  margin-left: 10px;
  cursor: pointer;
  background: none transparent;
}

.c3 .artist-follow:before {
  font-family: "artsy-icons";
  content: "î˜©";
  vertical-align: -8.5px;
  line-height: 32px;
  font-size: 32px;
}

.c3 .artist-follow:after {
  content: "Follow";
  font-family: 'Adobe Garamond W08', 'adobe-garamond-pro', 'AGaramondPro-Regular', 'Times New Roman', 'Times', 'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 17px;
  line-height: 1.1em;
  text-transform: none;
}

.c2 {
  margin-bottom: 15px;
  font-family: Unica77LLWebRegular , 'Arial', 'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 34px;
  line-height: 1.1em;
}

@media only screen and (min-width:0em) {
  .c1 {
    -webkit-flex-basis: 100%;
    -ms-flex-basis: 100%;
    flex-basis: 100%;
    max-width: 100%;
    display: block;
  }
}

@media only screen and (min-width:75em) {
  .c1 {
    -webkit-flex-basis: 33.333333333333336%;
    -ms-flex-basis: 33.333333333333336%;
    flex-basis: 33.333333333333336%;
    max-width: 33.333333333333336%;
    display: block;
  }
}

@media only screen and (min-width:64em) {
  .c1 {
    -webkit-flex-basis: 33.333333333333336%;
    -ms-flex-basis: 33.333333333333336%;
    flex-basis: 33.333333333333336%;
    max-width: 33.333333333333336%;
    display: block;
  }
}

@media only screen and (min-width:48em) {
  .c1 {
    -webkit-flex-basis: 33.333333333333336%;
    -ms-flex-basis: 33.333333333333336%;
    flex-basis: 33.333333333333336%;
    max-width: 33.333333333333336%;
    display: block;
  }
}

@media only screen and (min-width:0em) {
  .c4 {
    -webkit-flex-basis: 100%;
    -ms-flex-basis: 100%;
    flex-basis: 100%;
    max-width: 100%;
    display: block;
  }
}

@media only screen and (min-width:75em) {
  .c4 {
    -webkit-flex-basis: 50%;
    -ms-flex-basis: 50%;
    flex-basis: 50%;
    max-width: 50%;
    display: block;
  }
}

@media only screen and (min-width:64em) {
  .c4 {
    -webkit-flex-basis: 50%;
    -ms-flex-basis: 50%;
    flex-basis: 50%;
    max-width: 50%;
    display: block;
  }
}

@media only screen and (min-width:48em) {
  .c4 {
    -webkit-flex-basis: 50%;
    -ms-flex-basis: 50%;
    flex-basis: 50%;
    max-width: 50%;
    display: block;
  }
}

@media (max-width:600px) {
  .c3 p,
  .c3 ul,
  .c3 ol {
    font-family: 'Adobe Garamond W08', 'adobe-garamond-pro', 'AGaramondPro-Regular', 'Times New Roman', 'Times', 'serif';
    -webkit-font-smoothing: antialiased;
    font-size: 19px;
    line-height: 1.5em;
  }

  .c3 li {
    font-family: 'Adobe Garamond W08', 'adobe-garamond-pro', 'AGaramondPro-Regular', 'Times New Roman', 'Times', 'serif';
    -webkit-font-smoothing: antialiased;
    font-size: 19px;
    line-height: 1.5em;
  }

  .c3 h1 {
    font-family: Unica77LLWebRegular , 'Arial', 'serif';
    -webkit-font-smoothing: antialiased;
    font-size: 34px;
    line-height: 1.1em;
  }

  .c3 h2 {
    font-family: Unica77LLWebRegular , 'Arial', 'serif';
    -webkit-font-smoothing: antialiased;
    font-size: 32px;
    line-height: 1.1em;
  }

  .c3 h3 {
    font-family: Unica77LLWebRegular , 'Arial', 'serif';
    -webkit-font-smoothing: antialiased;
    font-size: 16px;
    line-height: 1.1em;
    line-height: 1.5em;
  }

  .c3 h3 strong {
    font-family: Unica77LLWebMedium , 'Arial', 'serif';
    -webkit-font-smoothing: antialiased;
    font-size: 16px;
    line-height: 1.1em;
  }

  .c3 blockquote {
    font-family: 'Adobe Garamond W08', 'adobe-garamond-pro', 'AGaramondPro-Regular', 'Times New Roman', 'Times', 'serif';
    -webkit-font-smoothing: antialiased;
    font-size: 34px;
    line-height: 1.1em;
  }

  .c3 .content-start {
    font-size: 55px;
  }
}

@media (max-width:900px) {
  .c2 {
    font-family: Unica77LLWebRegular , 'Arial', 'serif';
    -webkit-font-smoothing: antialiased;
    font-size: 32px;
    line-height: 1.1em;
  }
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <div
      className="c2"
    >
      Credits
    </div>
    <div
      className="article__text-section c3"
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "<p><b>Director</b></br>Marina Cashdan<br><b>Featuring</b></br>Trevor Paglan</p>",
          }
        }
      />
    </div>
  </div>
  <div
    className="c4"
  >
    <div
      className="c2"
    >
      About the Film
    </div>
    <div
      className="article__text-section c3"
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "<p>Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Curabitur blandit tempus porttitor. Donec ullamcorper nulla non metus auctor fringilla. Donec ullamcorper nulla non metus auctor fringilla. Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor. Maecenas faucibus mollis interdum. Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p><p>Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit. Curabitur blandit tempus porttitor. Sed posuere consectetur est at lobortis. Praesent commodo cursus magna, vel scelerisque nisl consectetur et. Nullam quis risus eget urna mollis ornare vel eu leo.</p><p>Donec sed odio dui. Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor. Nullam quis risus eget urna mollis ornare vel eu leo. Nulla vitae elit libero, a pharetra augue.</p>",
          }
        }
      />
    </div>
  </div>
</div>
`;
